src/shipit_static_analysis/shipit_static_analysis/workflow.py

```diff
@@ -144,8 +139,9 @@ def run(self, revision):
                 logger.info('No clang files detected, skipping mach')
 
             # Setup python environment
+            # --setup has been removed, --list still triggers the setup
```

Comment:
I would remove this comment, can be confusing for other people looking at this in the future who are not going to know `--setup` ever existed in the first place.